src/Illuminate/Routing/RouteDependencyResolverTrait.php

```diff
@@ -74,19 +87,68 @@ public function resolveMethodDependencies(array $parameters, ReflectionFunctionA
      * @param  array  $originalParameters
      * @return mixed
      */
-    protected function transformDependency(ReflectionParameter $parameter, $parameters, $originalParameters)
+    protected function transformDependency(ReflectionParameter $parameter, $parameters)
```

Comment:
You can add type hinting for `$parameters`, so it will be: `protected function transformDependency(ReflectionParameter $parameter, array $parameters)`